modules/olm-deprecations-schema.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+[id="olm-deprecations-schema_{context}"]
+= olm.deprecations schema
+
+The optional `olm.deprecations` schema defines deprecation information for packages, bundles, and channels in a catalog. Operator authors can use this schema to provide relevant messages about their Operators, such as support status and recommended upgrade paths, to users running those Operators from a catalog.
+
+An `olm.deprecations` schema entry can contain at least one of the following `reference` types, which indicates the deprecation scope. After the Operator is installed, any deprecation messages can be viewed as status conditions on the related `Subscription` object.
```

Comment:
"deprecation messages" 
Do we need to be more explicit that it's `entries[].message` displayed as an opaque blob in status conditions for the subscription?